setup/azurecmdfiles.wxi

```diff
@@ -759,15 +759,75 @@
                         <Component Id="cmpB2597D237191FA5EFBD36804D217035B" Guid="*">
                             <File Id="fil943575A5C76A6FE22FF176958FCBD7FD" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\AzureRM.MachineLearningCompute\AzureRM.MachineLearningCompute.psm1" />
                         </Component>
+                        <Component Id="cmp8DCB9196E6A234B735C6511571A39F87" Guid="*">
```

Comment:
@shutchings it looks like these extra dlls didn't get deleted during the full build. There is a `CleanupBuild.ps1` script that runs at the end of `msbuild build.proj` that removes these redundant assemblies in your output folder. Would you mind regenerating this file after a full build?